packages/core/nest-factory.ts

```diff
@@ -27,6 +30,12 @@ import { NestApplication } from './nest-application';
 import { NestApplicationContext } from './nest-application-context';
 import { DependenciesScanner } from './scanner';
 
+type IEntryNestModule =
```

Comment:
```suggestion
export type IEntryNestModule =
```
We have implemented a wrapper around `NestFactory.create()` and need to copy this type definition, because we can't reference it as it's not exported.

Reply:
we could export it. Not sure how to call it property tho

you can also do:

```ts
type NestFactoryFirstArg = Parameters<typeof NestFactory.create>[0];
```

Reply:
Thanks for the suggestion! 👍 
I'll use the type above for now and will try to open a PR next week.